### PR TITLE
Add new JS rules

### PR DESCRIPTION
**Pull request by inkz**
new rules for Express.js pack

### CODE REVIEW

File: javascript/express/security/audit/remote-property-injection.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+rules:
+- id: remote-property-injection
+  message: |
+    Bracket object notation with user input is present, this might allow an attacker to access all properties of the object and even it's prototype. Use literal values for object properties.
+  metadata:
+    owasp: 'A1: Injection'
+    cwe: 'CWE-400: Uncontrolled Resource Consumption'
```
Comment (minusworld) [inline javascript/express/security/audit/remote-property-injection.yaml]: Is this CWE correct?

Comment (inkz) [reply javascript/express/security/audit/remote-property-injection.yaml]: good question

Comment (minusworld) [reply javascript/express/security/audit/remote-property-injection.yaml]: How about CWE-1321: Improperly Controlled Modification of Object Prototype Attributes ('Prototype Pollution')?

Comment (inkz) [reply javascript/express/security/audit/remote-property-injection.yaml]: yeah lets change it to 1321

Comment (inkz) [reply javascript/express/security/audit/remote-property-injection.yaml]: Thank you :pray:

**Review by minusworld — APPROVED** [review]
lgtm!

**Pull request merged**